.travis.yml

```diff
@@ -24,6 +24,18 @@ matrix:
       dist: xenial
       sudo: true
       env: ANSIBLE_VERSION='ansible>=2.3,<2.4'
+    - os: linux
+      dist: bionic
```

Comment:
As far as I took a look on: https://docs.travis-ci.com/user/reference/overview/ - this OS, same as Debian are not supported by Travis.

Reply:
I don't think we have any direct impact on that CI, except making issue/request to them. Best would be to remove them from tests, if they're just running default image, which is tested anyway.